src/components/sidebar/left-sidebar/TotalStats.tsx

```diff
@@ -1,9 +1,31 @@
-import React from "react";
+import React, { useContext } from "react";
+import { AppContext } from "../../../context";
+import { aggregateRecords, getAggregateData } from "../../../metaAnalysis";
+import { Statistic } from "semantic-ui-react";
+import './TotalStats.css'
 
 export default function TotalStats() {
+  const [state, dispatch] = useContext(AppContext);
+
+  const { seroprevalence, n } = aggregateRecords(state.filtered_records);
+  const aggregatedData = getAggregateData(state.filtered_records, 'country');

```

Comment:
This is an unnecessary computation, there's probably a less wasteful way to compute the number of countries.

Maybe we can leverage some of the filters work to do this better (where we will likely store all country names in our context). If we decide to go this route, do you mind putting a TODO comment just so we don't forget about this?